index.d.ts

```diff
@@ -39,6 +39,13 @@ declare namespace globby {
 		@default false
 		*/
 		readonly gitignore?: boolean;
+
+		/**
+		Glob pattern to look for ignore files, which are then used to ignore globbed files.
```

Comment:
The definition of what "ignore files" are is not clear. The docs need to be expanded. There are also many unanswered questions: How does this relate to `gitignore` option. Does it work in addition to it, instead of it, etc.

You also need to add docs to the readme.

Reply:
Added a quick explanation here, and a larger explanation to the readme.